docs/snippets/examples/AcmeObservationCollector.java

```diff
@@ -0,0 +1,20 @@
+package examples;
+
+import io.smallrye.stork.api.observability.ObservationCollector;
+import io.smallrye.stork.api.observability.StorkEventHandler;
+import io.smallrye.stork.api.observability.StorkObservation;
+
+public class AcmeObservationCollector  implements ObservationCollector {
+    private static final StorkEventHandler ACME_HANDLER = ev -> {
+    };
```

Comment:
Here mention what needs to happen in the implementation.
That's where you get the composite (terminal event)